js/src/apbct-public--2--public.js

```diff
@@ -1415,11 +1415,33 @@ function apbctEmailEncoderCallbackBulk(result, encodedEmailNodes, clickSource) {
                         currentResultData = row;
                     }
                 });
-                // change text
+
                 let email = currentResultData.decoded_email.split(/[&?]/)[0];
+
+                // copy icon
+                const copyIcon = document.createElement('span');
+                copyIcon.classList.add('copy-email-icon');
+                copyIcon.innerHTML = '📋'; // You can replace this with an actual icon if needed
+                copyIcon.style.cursor = 'pointer';
+                copyIcon.title = 'Copy email to clipboard';
```

Comment:
Please, make this phrase translatable.

Reply:
You are right, it will be realized in the feature.